src/mmrelay/meshtastic/events.py

```
@@ -311,6 +311,12 @@ def on_lost_meshtastic_connection(
         facade.meshtastic_client = None
         facade._relay_active_client_id = None
 
+        pending_probe_timer = facade._pending_connect_time_probe_timer
+        facade._pending_connect_time_probe_timer = None
+        if pending_probe_timer is not None:
+            with contextlib.suppress(Exception):
```

Ruff (0.15.10)

[error] Undefined name `contextlib`

(F821)